### PR TITLE
Datepicker navigation button color fix

### PR DESCRIPTION
**Pull request by Darrken**
This fixes a small background-color issue with the datepicker. It's small enough it could be merged in along with another PR so as not to bump the version number. I also adjusted some syntax in the package.json that was breaking commits when using SourceTree.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "build": "babel src -d lib --ignore specs.js,*.md --copy-files",
     "ci": "npm run lint && npm test",
     "lint": "eslint src",
-    "precommit": "lint-staged && ./rebuild-docs.sh",
```
Comment (aabenoja) [inline package.json]: Apparently we need this for windows users. Should we replace this with `sh rebuild-docs.sh` instead for cross compatibility?

Comment (Darrken) [reply package.json]: Done.

Comment (aabenoja) [reply package.json]: No issues with SourceTree?

Comment (Darrken) [reply package.json]: Nope, worked fine.

**Pull request merged**